cmd/gce-pd-csi-driver/main.go

```diff
@@ -63,6 +63,7 @@ var (
 	waitForOpBackoffJitter    = flag.Float64("wait-op-backoff-jitter", 0.0, "Jitter for wait for operation backoff")
 	waitForOpBackoffSteps     = flag.Int("wait-op-backoff-steps", 100, "Steps for wait for operation backoff")
 	waitForOpBackoffCap       = flag.Duration("wait-op-backoff-cap", 0, "Cap for wait for operation backoff")
+	deviceInUseTimeout        = flag.Duration("device-in-use-timeout", 30, "Max time to wait for a device to be unsed when attempting to unstage")
```

Comment:
This is meant to be 30s, right? I think an integer is interpreted as a 30 nanoseconds.

Also, what do you think about adding a feature flag to default disable the device in use check? As this feature adds a memory datastructure it may impact total memory usage. It may be useful to be able to disable the device in use check entirely.

Reply:
Done. All deviceInUse.* functions will short-circuit if the feature is disabled.